server/views/commit/index.jade

```diff
@@ -28,8 +28,14 @@ block content
                                 input(type='radio' id=frontEndCert value=frontEndCert name='goal' checked="checked")
```

Comment:
Above, change to:
`.btn-group.btn-group-vertical(data-toggle='buttons' role='group')`